lib/src/RedisRemoteSaiInterface.cpp

```diff
@@ -903,19 +903,30 @@ sai_status_t RedisRemoteSaiInterface::waitForQueryAttributeCapabilityResponse(
             create_implemented, set_implemented, get_implemented);
 
         if(create_implemented)
+        {
```

Comment:
you could use 
capability->create_implemented = (create_implemnted) ? true : false;

it would save like 20 lines of code

Reply:
As part of the change to not use "number string" this part of the code has changed accordingly and it did simplify the coding.  please review my last change for this.